### ISSUE TITLE
Nginx Welcome Page after successful installation

### ISSUE THREAD

**Issue by zedrickvillas** [carlalexander/debops-wordpress#114]
After successful installation:
The site only display the Nginx Welcome page.

> Welcome to nginx!
> 
> If you see this page, the nginx web server is successfully installed and working. Further configuration is required.
> 
> For online documentation and support please refer to nginx.org.
> Commercial support is available at nginx.com.
> 
> Thank you for using nginx.

**Comment by carlalexander** [comment]
I've seen this issue before. I need to remember what caused it.

**Comment by zedrickvillas** [comment]
Yes, i've seen the ticket but the author didn't put the reason and the resolution. :(

**Comment by carlalexander** [comment]
There's this [issue](https://github.com/carlalexander/debops-wordpress/issues/89) that mentions it too. He just ran the `wordpress` playbook a second time I think.

**Comment by zedrickvillas** [comment]
Yes. I ran the wordpress playbook twice.
The first time the fail2ban error appeared, the second time it was successful, but the nginx welcome page was displayed when trying to access the site

**Comment by zedrickvillas** [comment]
Was now able to access the site. 
My hunch is right, i need to restart the server haha.
You may add it on the steps, to be sure. 
Closing this ticket.

**Comment by ejholdings** [comment]
TLDR; if `$ debops wordpress`  ran successfully (esp after requiring more than one try), restart your server and your website will likely load.

Full Description:
In case anyone else has the same problem -- the Nginx welcome page appearing after a successful installation, I did too.  I'm using a Digital Ocean VM (Ubuntu 16.04).  Upon running `$ debops wordpress` the script ran for 10 - 15 minutes and then failed due to fail2ban not restarting.  I ran `$ debops wordpress` a second time  (based on another thread on this git project) and didn't encounter any problems with the installation of Wordpress.  When the Wordpress installation finished I could only get the Nginx welcome page.  I restarted my VM and the website loaded as expected immediately.
